src/Server.zig

```diff
@@ -376,6 +376,7 @@ fn initializeHandler(server: *Server, _: std.mem.Allocator, request: types.Initi
         log.info("client is '{s}-{s}'", .{ clientInfo.name, clientInfo.version orelse "<no version>" });
 
         if (std.mem.eql(u8, clientInfo.name, "Sublime Text LSP")) blk: {
+            server.config.completion_label_details = false;
```

Comment:
Any editor that supports label details will struggle to display long label details with a small enough editor window, not just Sublime Text.
What is the motivation for having this option be `false` only in Sublime Text?

The default value of every config option is documented. Secretly overriding the default value on a specific editor/client should be clearly documented so that the user is not being misled.

Overriding the default value on a specific editor/client also means that we are interfering with how the client chooses to display the information we provide them. Ideally, the server/ZLS should behave identical on all clients (workarounds are an exception).

Reply:
>  What is the motivation for having this option be false only in Sublime Text?

Apologies, I had a comment in the option docstring stating it would be set to `false` in Sublime by default but I forgot to include that part when I resubmitted the PR. 

The motivation for defaulting to `false` in Sublime is due to how the completion results are rendered and the majority of return types aren't visible. I believe it will be a better experience for Sublime users. That's just my opinion, it was annoying enough to me to implement this change. That said, happy to change the PR to either a `true` default for all editors or improve the documentation to explain why `false` is the default for Sublime.

Reply:
I also now see there is no way to override `server.config.completion_label_details = false;` in `zls.json`. My original intention was to default to `false` but allow the user to change it back to `true` if desired.

Reply:
I do see a reason for having this disabled on Sublime Text, but the same still goes for VSCode when you use a small enough window. Though, I've observed that VSCode was better at using the limited window space.

I would suggest to change it to `true` for all editors since this option tries to resolve an issue on the editor side which 
ideally shouldn't be the responsibility of the server/ZLS to work around. Perhaps, how the ST LSP plugin (or other clients) deals with long completion label details could be brought to their attention to discuss further improvements.

Reply:
Sounds good. I removed the line forcing `false` for Sublime Text. I've also mentioned this to ST LSP maintainers. Thanks.

Reply:
From my cursory glance, is this a server setting? If so, you can set server settings via the client configuration json in ST LSP.

Reply:
> From my cursory glance, is this a server setting? If so, you can set server settings via the client configuration json in ST LSP.

Yes this is a server setting, since it has value in vscode and at in Neovim (at least in the Lazy.nvim config I use). Good to know it can be set on the client, thanks.